src/main/java/interface_adapter/login/club_login/ClubLoginPresenter.java

```diff
@@ -43,10 +43,16 @@ public void prepareSuccessView(ClubLoginOutputData response) {
      * @param response the input data getting passed to the presenter
      */
     private void setHomePageState(ClubLoginOutputData response) {
-        final ClubHomeState clubHomeState = clubHomeViewModel.getState();
-        clubHomeState.setUsername(response.getUsername());
-        this.clubHomeViewModel.setState(clubHomeState);
-        this.clubHomeViewModel.firePropertyChanged();
+        final ClubLoggedInState clubHomeState = clubLoggedInViewModel.getState();
+        clubHomeState.setClubName(response.getUsername());
+        clubHomeState.setEmail(response.getEmail());
+
+        // IMPORTANT!!!
+        // Since we are changing the code to have a get posts use case, we don't need to assign it to the
+        // ClubLoggedInState. Therefore, this logic will be taking place in the View code.
```

Comment:
Just for clarification, the "get posts" use case and it's presenter would be the one to get the updated list of members, so the logic is technically in the use case, not view. Let me know if there is still some confusion on how the view is updated!

Reply:
Yes. That is what we agreed on yesterday, however I forgot to update this comment. I will make sure to change this, thanks! If I have any questions, I'll be sure to ask!